app/javascript/packs/stylesheets/bootstrap.scss

```diff
@@ -290,3 +290,6 @@ select.form-control {
   margin-top: -80px;
 }
 
+.red-icon {
+  color: red;
+}
```

Comment:
Not that this is a HUGE thing or anything, but this probably belongs in a different `scss` file. 
I just checked our styling files, and none of them look like an awesome fit. Maaaaybe `monitoree_details`...? To be honest, I'd make a new one called like `patient.scss` or something.

Reply:
Sure, I can look to create a new file for this styling.

Reply:
Addressed in [BAF4994](https://github.com/SaraAlert/SaraAlert/pull/745/commits/baf49948ca64a2c6eb3865faf010f2404e5d9b0d)